examples/merkle-tree-cli/README.md

```diff
@@ -0,0 +1,78 @@
+<div align="center">
+
+# Lambdaworks Merklee Tree CLI
+
+Simple Merkle Tree CLI that uses [Poseidon hash](https://www.poseidon-hash.info/).
+
+</div>
+
+### Usage:
+
+**To create a Merkle Tree and save it to a file you can use:**
+
+```bash
+cargo run --release generate-merkle-tree <TREE_PATH>
+```
+
+For example:
+
+```bash
+cargo run --release generate-merkle-tree sample_tree.csv
+```
+
+
+**To generate proof for a Merkle Tree you can use:**
+
+```bash
+cargo run --release generate-merkle-tree <TREE_PATH> <POSITION>
```

Comment:
this should call the command `generate-proof`

Reply:
should the file be the original `csv` or the `json` tree?

Reply:
Oh, I see that was mentioned in the PR description, sorry

Reply:
> this should call the command `generate-proof`

Correct, modified.

> should the file be the original csv or the json tree?

Which file do you think is the better option?

Reply:
I think the original csv seems a better option

Reply:
Leaving as is then.